setup.py

```diff
@@ -56,11 +56,14 @@
         # needed to process accounting data
         'numpy >= 1.14.0',
         'pandas >= 0.23.0',
+        'pandas >= 0.23.0, < 1.0.0',
```

Comment:
can you add a comment on why the upper bound is needed?

Reply:
comment added in https://github.com/sisc-hpc/vsc-accounting-brussel/pull/2/commits/0e6b8be9e5067f77ad7864527d26e2d23ba63b88